webapp/crashstats/settings/base.py

```diff
@@ -41,12 +50,20 @@ def path(*dirs):
         os.environ[key] = val
 
 
-# Whether or not we're running in the local development environment
-LOCAL_DEV_ENV = config("LOCAL_DEV_ENV", False, cast=bool)
+LOCAL_DEV_ENV = config(
+    "LOCAL_DEV_ENV",
+    default="false",
+    parser=parse_bool,
+    doc="Whether or not we're running in the local development environment",
+)
 
-# Debugging displays nice error messages, but leaks memory. Set this to False
-# on all server instances and True only for development.
-DEBUG = config("DEBUG", False, cast=bool)
+DEBUG = config(
+    "DEBUG",
+    default="false",
+    parser=parse_bool,
+    doc="Debugging displays nice error messages, but leaks memory. Set this to false"
+    " on all server instances and true only for development.",
```

Comment:
I would put this in parentheses so it's easier to see that this is two strings concatenated by the Python parser into a single string.

```suggestion
    doc=(
        "Debugging displays nice error messages, but leaks memory. Set this to false"
        " on all server instances and true only for development."
    ),
```

I also put the whitespace at the end of the line, so this looks different than how I've been doing it. However, I'm not sure that matters, so I wouldn't change it.

Vaguely related to this, but not really--one thing to note here is that Everett will pick up a single string where the concatenation is implied and handled by the Python parser like this, but it won't pick up a string where the concatenation is done at runtime with the `+` operator. So what you've got works and we shouldn't change it to something like `"string 1" + "string 2"`. That's a bug in Everett that I keep meaning to write up an issue for. I did that just now: https://github.com/willkg/everett/issues/240